CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## HEAD
```

Comment:
@kipcole9 fyi, I'll leave this on master for a few days, I'm thinking about incorporating [1] change before next release.

If however, you'd like a release sooner than later to update your stuff, lmk!

[1] https://github.com/wojtekmach/calendar_interval/pull/6/files#r323873568

Reply:
No rush @wojtekmach, I’ll work on deleting my not-very-good interval module in `ex_cldr_calendars` and replacing it with yours.  But there’s no immediate need.  I’ll think about the `interval in other_interval` issue as well now that my head is past this part.